spread/project.go

```diff
@@ -64,6 +64,10 @@ type Backend struct {
 	Location string
 	Storage  Size
 
+	// Only for Openstack so far
+	Network        string
+	SecurityGroups []string `yaml:"security-groups"`
```

Comment:
Just as a datapoint, this can be any string, we have seen things like `juju-054c6083-a7ba-4e30-9c51-c2c58c74207d`  or `default`.

Reply:
The feedback from Gustavo for this one is that the more generic term `groups:` should be used here and that is then applied in openstack to security-groups and in GCE to the equivalent concept (VPC?)